lib/switchtec.c

```diff
@@ -986,7 +1094,8 @@ static int log_a_to_file(struct switchtec_dev *dev, int sub_cmd_id,
 
 			/* parse the log data and write it to a file */
 			ret = write_parsed_log(res.data, res.hdr.count,
-					       entry_idx, &defs, log_file);
+					       entry_idx, &defs, true,
```

Comment:
Don't use true/false when the argument takes an enum. Use actual enum values.

Reply:
Same as the above -- I forgot to update it to the enum.